components/webgpu/lib.rs

```diff
@@ -16,7 +16,7 @@ use smallvec::SmallVec;
 
 #[derive(Debug, Deserialize, Serialize)]
 pub enum WebGPUResponse {
-    RequestAdapter(String, WebGPUAdapter),
+    RequestAdapter(String, WebGPUAdapter, WebGPU),
```

Comment:
Could it make sense for the `WebGPU` to keep a sender to itself, so that the constellation doesn't have to send it in the request?

Reply:
I think yes. Updated the code according to that.